src/quo2/foundations/colors.cljs

```diff
@@ -133,6 +133,11 @@
 (def neutral-95-opa-70-blur neutral-95-opa-70)
 (def neutral-100-opa-70-blur neutral-100-opa-70)
 
+;;;;Blur-IOS
+;; https://github.com/status-im/status-mobile/issues/14903
+(def neutral-80-opa-80-blur-ios "#1E2430CC")
```

Comment:
Perhaps we can just use the same value but a platform check in the color? Wdyt? It could still be a def too because the check is only needed once, right?

Reply:
This looked safer choice, because `neutral-80-opa-80-blur` also used at few other places. Not sure we have similar problems at those places too.

Reply:
Do we know when we should be using this color and not the other one?

Perhaps we can make the name specifics by the use case. E.g

Reply:
Actually I was opting for safe side, because differences are very hard to notice.

If you see in below images, right one is little bluish
<div align="center">
<img src="https://github.com/status-im/status-mobile/assets/17097240/c83c1111-b00f-4111-ae0b-70ad14c44607" alt="Screenshot" height="400"/>
<img src="https://github.com/status-im/status-mobile/assets/17097240/b100fa74-1092-47e0-9489-52f7c09cbfd5" alt="Screenshot" height="400"/>
</div>

But I checked the design and in figma buttons are not bluish, so I think you are right. Its probably safe to just replace uses of blur color completely.